core/docker/default/etc/jvm.config

```diff
@@ -12,3 +12,5 @@
 -XX:PerBytecodeRecompilationCutoff=10000
```

Comment:
In the commit message, please add reference to original JDK issues and backport PR. You should also mention why it's OK to enable